ios/Podfile

```diff
@@ -42,10 +75,14 @@ target 'Mattermost' do
   pod 'XCDYouTubeKit', '2.8.2'
   pod 'Swime', '3.0.6'
 
-  target 'MattermostTests' do
```

Comment:
Removed the `MattermostTests` target, not really used and was causing more than 100 different errors after the upgrade